crates/types/src/network.rs

```diff
@@ -116,13 +108,7 @@ impl Network {
         //Provider::new(client)
     }
 
-    pub async fn reset_listener(&self) -> Result<()> {
-        if self.is_dev().await {
-            let http = Url::parse(&self.http_url)?;
-            let ws = Url::parse(&self.ws_url())?;
-            ethui_broadcast::reset_anvil_listener(self.chain_id, http, ws).await;
-        }
-
+    pub async fn reset_listener(&self) -> Result<(), RpcError<TransportErrorKind>> {
```

Comment:
This is a temporary change just to make the app compile. It's necessary to do so as moving this here adds a cyclic dependency with `ethui_broadcast`

Reply:
I'm ok with it :+1: